data/map planets.txt

```diff
@@ -2504,6 +2550,25 @@ planet Luna
 		threshold 7500
 		fleet "Large Republic" 10
 
+planet "Mordente-Bridi"
+	attributes ccor military station urban
+	landscape land/city6
+	music ambient/machinery
+	description `A testament to the ingenuity of the engineering corps of the now defunct CCOR, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to that of a sizable city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
+	spaceport `Apart from the spaceport facilities, almost none of Mordente Cylinder is pressurized. Making up most of its armored hull is a micro-gravity shipyard, which despite not being subject to a central government and defense force has remained quite busy. Worker shuttles are busily flying about the large space, tending to a multitude of ships; you notice that a few of them are also engaged in constructing new vessels.`
```

Comment:
Bridi Cylinder and Mordente Cylinder? Are they city or spaceport or whatever? Mb it could be specified the first time they're mentionned, just above, and here?